container/label/label.go

```diff
@@ -12,7 +12,7 @@ type LabelContainer struct {
 	Metric string `label:"kappa.metric"`
 }
 
-func (lc LabelContainer) String() string {
+func (lc *LabelContainer) String() string {
```

Comment:
A pretty print function should not be able to modify the LabelContainer and propagate that to the main instance